### ISSUE TITLE
Sphinx Design

### ISSUE THREAD

**Issue by kolibril13** [ManimCommunity/manim#309]
Hi, I will now start to test some sphinx designs and share them in this thread.
Therefore I will make two screenshots for each design (scree resolution of 2560 x 1440), one of an example page, one of a function from the reference page.
For each theme, I will post a separate comment, so please wait a little bit before you answer in this issue.

**Comment by kolibril13** [comment]
# sphinx_rtd_theme
![image](https://user-images.githubusercontent.com/44469195/90332044-cc6f2000-dfb9-11ea-810e-ce28be114852.png)
![image](https://user-images.githubusercontent.com/44469195/90332046-d09b3d80-dfb9-11ea-8976-294cbd8b0c67.png)

**Comment by kolibril13** [comment]
# sphinx_modern_theme
![image](https://user-images.githubusercontent.com/44469195/90332165-10aef000-dfbb-11ea-89b2-0a1fc88cedf7.png)
![image](https://user-images.githubusercontent.com/44469195/90332167-14427700-dfbb-11ea-90fa-1fae35b53800.png)

**Comment by kolibril13** [comment]
# guzzle_sphinx_theme
![image](https://user-images.githubusercontent.com/44469195/90332319-2375f480-dfbc-11ea-923e-cda5191a3b23.png)
![image](https://user-images.githubusercontent.com/44469195/90332320-27097b80-dfbc-11ea-9357-0be0afe35509.png)

**Comment by kolibril13** [comment]
#sphinx_modern_theme
![image](https://user-images.githubusercontent.com/44469195/90332394-c6c70980-dfbc-11ea-80a0-32e8b58a78c6.png)
![image](https://user-images.githubusercontent.com/44469195/90332398-cc245400-dfbc-11ea-8873-be235585d3c5.png)

**Comment by kolibril13** [comment]
# python_docs_theme
![image](https://user-images.githubusercontent.com/44469195/90332563-270a7b00-dfbe-11ea-8694-7f144bb30190.png)
![image](https://user-images.githubusercontent.com/44469195/90332566-2a9e0200-dfbe-11ea-9374-9614aba77c29.png)

**Comment by kolibril13** [comment]
# sphinx_fossasia_theme
![image](https://user-images.githubusercontent.com/44469195/90332630-cfb8da80-dfbe-11ea-8c0f-f3d284b5f252.png)
![image](https://user-images.githubusercontent.com/44469195/90332632-d34c6180-dfbe-11ea-94c5-8462734ac343.png)

**Comment by kolibril13** [comment]
Ok, this is something to get started with.
By the way:
The black background of the scenes is sticking out for me a little to much.
I tested some other, maybe a little more decent background colors, what would you say to these:
1. original: Black
![image](https://user-images.githubusercontent.com/44469195/90332852-c466ae80-dfc0-11ea-9123-99c33104e141.png)

2. #2B2B2B (from the pycharm dracula colortheme)
![c#2B2B2B](https://user-images.githubusercontent.com/44469195/90332868-dd6f5f80-dfc0-11ea-9d09-c5c0d44c5a0b.png)

3. #1D2126 (from an online dark theme)
![c#1D2126](https://user-images.githubusercontent.com/44469195/90332884-f2e48980-dfc0-11ea-8be4-662f3d7add60.png)
And number 3 in action on the website:
![image](https://user-images.githubusercontent.com/44469195/90332894-127bb200-dfc1-11ea-8f8e-f4a36405d607.png)

**Comment by leotrs** [comment]
I like guzzle theme. I think it's a good idea to choose some some colors for our examples, as long as all of them use the same color palette. That will give some consistency and a totally different vibe to the site.

**Comment by eulertour** [comment]
I have a slight preference for the RTD theme, though it's mostly because the others don't have much color.

**Comment by naveen521kk** [comment]
Whatever theme we chose we choose, I consider we would need some JavaScript work for adding some lazy loading things(like pictures and gifs) for the site to be fast enough because our documentation is going to be filled with pictures and videos. :)

**Comment by kolibril13** [comment]
> I have a slight preference for the RTD theme, though it's mostly because the others don't have much color.

For me the opposite is the case: I don't like the rtd so much because of the color.
I would prefer a design that is plain, with maybe some small accents, but the color will come from the images and videos.

> Whatever theme we chose we choose, I consider we would need some JavaScript work for adding some lazy loading things(like pictures and gifs) for the site to be fast enough because our documentation is going to be filled with pictures and videos. :)

Sounds good, but also a bit of work to implement. 
When we use javascript, would it then be possible to have the gifs/videos on pause by default, and make them move as soon as you hover over them with the mouse? Just like at the youtube front page.
Sometimes I find it a little strenuous  to have lots of moving gifs on the screen  (just like [Homer's website](https://www.youtube.com/watch?v=ae5UXhUtxao))

**Comment by naveen521kk** [comment]
> When we use javascript, would it then be possible to have the gifs/videos on pause by default, and make them move as soon as you hover over them with the mouse? Just like at the youtube front page.

Everything is possible. But I don't think standard themes allow us to changes these JavaScripts directly. If they allow it is good. Or else, we can make fork out of it and customize it. :)

**Comment by leotrs** [comment]
Yes, we will most likely need to have a custom theme in the future.

I'm neutral about whether the color comes from the examples only or the theme itself. But what I do not like from the ones that are really plain is that the documentation is hard to read for me. For example, in the second screenshot of python_docs_theme, there are no font colors or accents anywhere and I can barely look at that page. Eww.

**Comment by leotrs** [comment]
Update: once #324 is merged, then all that stands between us and a dedicated site for documentation (for example on ReadTheDocs) is this issue.

I suggest we quickly choose one of the themes posted by @kolibril13 and use that for the time being. After all, not a ton of people would be visiting the RTD website as soon as that's up. We can keep tweaking this theme as we approach a release version.
